x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/similarity_parameter.tsx

```diff
@@ -23,7 +23,7 @@ export const SimilarityParameter = ({ defaultToggleValue }: Props) => (
       defaultMessage: 'Set similarity',
     })}
     description={i18n.translate('xpack.idxMgmt.mappingsEditor.setSimilarityFieldDescription', {
-      defaultMessage: 'Which scoring algorithm or similarity should be used.',
+      defaultMessage: 'The scoring algorithm or similarity to use.',
```

Comment:
Since there are only two algorithms available, what do you think of simplifying this parameter so it's just the toggle?

![image](https://user-images.githubusercontent.com/1238659/71038971-eaf1e400-20d7-11ea-9efa-5b02d59c37f1.png)

> Use boolean similarity scoring algorithm

> This is best used when full-text ranking is not needed and relevancy score should only be based on whether the query terms match.

Reply:
Do you think by making this change we're overemphasizing the non-default value? Also, do you think it's worth mentioning what the default behavior is?

Reply:
I just looked at the [7.5 docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/similarity.html) and now there are three options: BM25, classic, and boolean. So I guess my suggestion here is moot and we should actually update the options to support all three. :)

Reply:
Yeah, I see that too. However, it doesn't seem to be valid.

```
PUT my_index
{
  "mappings": {
    "properties": {
      "keyword_field": {
        "type": "keyword",
        "similarity": "classic" 
      }
    }
  }
}
```

`"Unknown Similarity type [classic] for field [keyword_field]"`